examples/reach.amp.html

```diff
@@ -0,0 +1,60 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8" />
+  <title>Reach Player Example</title>
+  <link rel="canonical" href="amps.html" >
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1" />
+  <link href='https://fonts.googleapis.com/css?family=Lato' rel='stylesheet' type='text/css'>
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+    <style amp-custom>
+        body {
+            background-color: #eee;
+            max-width: 527px;
+            margin: 10px 10px 10px 10px;
+            font-family: 'Lato', Arial;
+        }
+        h1,h2,h3 {
+            margin: 10px auto 10px auto;
+        }
+        button {
+            padding: 12px 0;
+            margin: 8px auto;
+            border: 2px solid #fff;
+            min-width: 200px;
+            font-weight: 600;
+            font-size: 1rem;
+        }
+        button:hover, button:active, button:focus {
+            background: #1abc9c;
+            border-color: #1abc9c;
+            color: white;
+        }
+    </style>
+  <script async custom-element="amp-reach" src="https://cdn.ampproject.org/v0/amp-reach-0.1.js"></script>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+</head>
+<body>
+
+  <h2>AMP-Reach Player Example</h2>
+
+  <p>
+      Reach is a next-generation video syndication platform
+      connecting content owners, publishers, and advertisers.
+  </p>
+
+  <amp-reach
```

Comment:
How about calling it `amp-reach-player`. I think reach may not be widely known enough to warrant this short tag name :)